app/pokemon/[pokemonIndex]/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { get } from '@/lib/request'
+import Image from 'next/image'
+import Link from 'next/link'
+
+export const PokemonPage = async ({ params }) => {
+  const { pokemonIndex } = params
+
+  const pokemon = await get(`${process.env.SECRET_API}/${pokemonIndex}`).then(
+    (res) => res
+  )
```

Comment:
its worth mentioning cache here, in < Next.js 15 using get is fetched by default. If the data is ever changed or even the user changes then there will be consistency issues. This is why we chose axios in the test but generally this wont change so its fine.